src/styles/lines/lines.js

```diff
@@ -9,13 +9,17 @@ import {buildPolylines} from '../../builders/polylines';
 import Geo from '../../geo';
 import Utils from '../../utils/utils';
 
+let fs = require('fs');
+let shaderSrc_polygonsVertex = fs.readFileSync(__dirname + '/../polygons/polygons_vertex.glsl', 'utf8');
```

Comment:
After we talked about this I realized that by including the `readFileSync` here, we are duplicating the strings for the polygon shaders (which was one of the motivations for them being shared, though not the only one). It's probably not a huge amount of extra weight, but we could avoid that by importing the `Polygon` style and referencing its shader source properties?